exercise/templates/exercise/_points_badge.html

```diff
@@ -6,27 +6,34 @@
 </span>
 
 {% elif confirm_the_level %}
-{% if submitted %}
-<span class="badge{% if graded and official %} badge-{% if full_score %}success{% elif passed %}warning{% else %}danger{% endif %}{% endif %} {{ classes }}">
-  {% if not official %}{% trans 'Submitted after deadline' %}{% elif passed %}{% trans "Accepted" %}{% elif graded %}{% trans "Rejected" %}{% else %}{% trans "Submitted" %}{% endif %}
-</span>
-{% elif confirmable_points %}
-<span class="badge badge-danger" data-toggle="tooltip" title="{% trans 'You must submit here to confirm your points. Submissions are accepted even after the deadline.' %}">
-  {% trans "Missing" %}
-</span>
-{% elif exercise_page %}
-<span class="badge">
-  {% trans "Not submitted" %}
-</span>
-{% endif %}
+	{% if submitted %}
+	<span class="badge{% if graded and official %} badge-{% if full_score %}success{% elif passed %}warning{% else %}danger{% endif %}{% endif %} {{ classes }}">
+	  {% if not official %}{% trans 'Submitted after deadline' %}{% elif passed %}{% trans "Accepted" %}{% elif graded %}{% trans "Rejected" %}{% else %}{% trans "Submitted" %}{% endif %}
+	</span>
+	{% elif confirmable_points %}
+	<span class="badge badge-danger" data-toggle="tooltip" title="{% trans 'You must submit here to confirm your points. Submissions are accepted even after the deadline.' %}">
+	  {% trans "Missing" %}
+	</span>
+	{% elif exercise_page %}
+	<span class="badge">
+	  {% trans "Not submitted" %}
+	</span>
+	{% endif %}
 
 {% else %}
-{% if difficulty %}
-<span class="difficulty">{{ difficulty }}</span>
-{% endif %}
-<span class="badge{% if submitted and official %} {% if full_score %}badge-success{% elif passed %}badge-warning{% else %}badge-danger{% endif %}{% if unconfirmed %} unconfirmed-points{% endif %}{% endif %}{% if classes %} {{ classes }}{% endif %}"
+	{% if difficulty and difficulty != "-" %}
+	<span class="difficulty">{{ difficulty }}</span>
+	{% endif %}
+<span class="badge{% if submitted and official %} {% if full_score %}badge-success{% elif passed %}badge-warning{% else %}badge-danger{% endif %}{% if unconfirmed %} unconfirmed-points{% endif %}{% endif %}{% if classes %} {{ classes }}{% endif %}
```

Comment:
Should split the old too long line to multiple lines as you are adding your additions to different lines.
Or in the other words, when your change makes difference to an old line, there is no harm to refactor that line at the same time.

Reply:
Btw. this line has different indentation than line before.